src/components/artists-container.jsx

```diff
@@ -0,0 +1,24 @@
+import React, { Component } from 'react'
+import { observer } from 'mobx-react'
+import ArtistsList from './artists-list'
+import ArtistInfo from './artist-info'
+import artistInfoStore from '../stores/artist-info-store'
+
+export default @observer class ArtistsContainer extends Component {
+  render() {
+    return (
+      <div>
+        <ArtistsList
+          artistsStore={this.props.artistsStore}
+          artistInfoStore={this.props.artistInfoStore}
+        />
+        {artistInfoStore.displayInfo ? (
```

Comment:
Hmm teoretycznie to nie powinno być w `artistInfoStore`, bo to nie jest cześć artist info. Idealnie to powinno zależeć od ściezki / route'a, ale nie masz tego mechanizmu w ogóle, więc I guess może zostać. Ewentualnie przeniósłbym to do stanu komponentu albo zrobić store z `app state`.

Reply:
bedzie dodawany react router, wiec wtedy postaram sie rozwiazac ten problem